app/views/unattended/report_templates/host_-_installed_packages

```diff
@@ -0,0 +1,29 @@
+<%#
+name: Host - All Installed Packages
+snippet: false
+template_inputs:
+- name: Hosts filter
+  required: false
+  input_type: user
+  description: Limit the report only to hosts found by this search query. Keep empty
```

Comment:
Looks good, one small changed:

`Limit the report to only hosts found by this search query`